src/main/java/stevie/StevieUi.java

```diff
@@ -0,0 +1,83 @@
+package stevie;
+
+import java.util.Scanner;
+
+/**
+ * This class interacts with the user by receiving inputs from the user, as well as, returning the
+ * response to the user.
+ */
+public class StevieUi {
+    private Scanner sc;
+
+    public StevieUi() {
+        sc = new Scanner(System.in);
+    }
+
+    private static String help() {
+        return "\"list\": to display your activities.\n" +
+                "\"bye\": to end our session.\n" +
+                "\"mark <i>\" to mark the i-th task as done.\n" +
+                "\"unmark <i>\" to unmark the i-th task as done.\n" +
+                "\"delete <i>\" to delete the i-th task.\n" +
+                "\"todo <task_name>\" to add a todo task.\n" +
+                "\"deadline <task_name> /by <date>\" to add a deadline.\n" +
+                "\"event <event_name> /at <date>\" to add an event.\n" +
+                "Date should in format of dd/mm/yyyy HH:mm";
```

Comment:
Maybe the '+' operator should be placed after the line break?
```
     return "\"list\": to display your activities.\n" 
                + "\"bye\": to end our session.\n" 
                + "\"mark <i>\" to mark the i-th task as done.\n" 
                + "\"unmark <i>\" to unmark the i-th task as done.\n" 
                + "\"delete <i>\" to delete the i-th task.\n" 
                + "\"todo <task_name>\" to add a todo task.\n" 
                + "\"deadline <task_name> /by <date>\" to add a deadline.\n"
                + "\"event <event_name> /at <date>\" to add an event.\n" 
                + "Date should in format of dd/mm/yyyy HH:mm";
```
The "Place line break to improve readability" of the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html) recommends line breaks to be done that way.

Reply:
Thanks for pointing that out! I did not notice that when reviewing my own code.